deploy/helm/templates/NOTES.txt

```diff
@@ -0,0 +1,10 @@
+You have installed starboard-operator in the namespace "{{ .Release.Namespace }}"
+and it is configured to operate in the namespaces: '{{ tpl .Values.targetNamespaces . | default "(all namespaces)" }}'.
+
+Inspect created VulnerabilityReports by:
+
+    kubectl get vulnerabilityreports --all-namespaces
+
+Inspect the work log of starboard-operator by:
+
+    kubectl logs -n {{ .Release.Namespace }} deploy/{{ include "starboard-operator.fullname" . }}
```

Comment:
I updated NOTES.txt to reflect that this Helm chart doesn't expose a service which is consumed publicly, but instead simply installs something that works against the k8s api-server and creates VulnerabilityReport resources.